lib/v2/macmenubar/recently.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+
+function renderDescription(item) {
+    const categories = item.category.map((raw) => raw.name);
+    const tags = item.tag.map((raw) => raw.name);
+    return art(path.join(__dirname, 'template/description.art'), {
+        title: item.title,
+        tags,
+        categories,
+        content: item.content,
+    });
+}
+
+module.exports = async (ctx) => {
+    const url = 'https://macmenubar.com/wp-json/wp/v2/posts?per_page=100';
+    const category = ctx.params.category;
+    const response = await got(url);
+    const items = response.data
+        .filter((post) => post.category_info.some((cat) => category === cat.slug))
```

Comment:
You can use `categories` search parameter in the API call to ask the remote server to return the selected articles instead of filtering it locally. 
The category id for `developer-apps` is `60` which can be obtained through `/wp-json/wp/v2/categories?slug=developer-apps`